pipenv/vendor/vendor.txt

```diff
@@ -2,8 +2,10 @@ click-didyoumean==0.3.1
 click==8.1.7
 colorama==0.4.6
 dparse==0.6.3
+importlib-metadata==6.5.1
+packaging==24.0
```

Comment:
Our own copy of pip has packaging 21.3. Will your code work with it too?

I

Reply:
It does not -- I had to do this but pip itself needs to use its own version as does pipenv check depends on the old version currently.   Its not ideal having two versions of packaging, but I think its a temporary solution and getting off pkg_resources will be big for us.